### PR TITLE
[FR] Update `Ranking_Criteria/Skin_set`

### PR DESCRIPTION
**Pull request by Biokax**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Ranking_Criteria/Skin_Set_List/fr.md [hunk 1]
```diff
@@ -37,13 +32,20 @@ L'ensemble de ce skin set est pertinent pour le gameplay.
...
-| lighting.png | - | Optionnel | - |
 | particle50.png | Doit être inclus avec toutes les autres particules ; ne fonctionnera que si des hitbursts personnalisés sont inclus | Optionnel | 7x7px |
 | particle100.png | Doit être inclus avec toutes les autres particules ; ne fonctionnera que si des hitbursts personnalisés sont inclus | Optionnel | 7x7px |
 | particle300.png | Doit être inclus avec toutes les autres particules ; ne fonctionnera que si des hitbursts personnalisés sont inclus | Optionnel | 7x7px |
 | sliderpoint10.png | Doit être inclus avec sliderpoint30.png ; ne fonctionnera que si le skin spécifique à l'utilisateur prend en charge le Skin V1 | Optionnel | - |
 | sliderpoint30.png | Doit être inclus avec sliderpoint10.png ; ne fonctionnera que si le skin spécifique à l'utilisateur prend en charge le Skin V1 | Optionnel | - |
 
+### Jeu de lumières
+
+L'ensemble de ce set de skin est pertinant pour le jeu.
```
Comment (DoryanR) [inline wiki/Ranking_Criteria/Skin_Set_List/fr.md:43]:
```suggestion
L'ensemble de ce skin set est pertinent pour le gameplay.
```

File: wiki/Ranking_Criteria/Skin_Set_List/fr.md [hunk 2]
```diff
@@ -37,13 +32,20 @@ L'ensemble de ce skin set est pertinent pour le gameplay.
...
+
+| Nom du fichier | Notes | Achèvement du set | Tailles recommandées pour les skins SD |
```
Comment (DoryanR) [inline wiki/Ranking_Criteria/Skin_Set_List/fr.md:45]:
```suggestion
| Nom du fichier | Notes | Pertinence de l'ajout | Tailles recommandées pour les skins SD |
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

**Review by DoryanR — APPROVED** [review]

File: wiki/Ranking_Criteria/Skin_Set_List/fr.md [hunk 3]
```diff
@@ -37,13 +32,20 @@ L'ensemble de ce skin set est pertinent pour le gameplay.
...
 | hit300g(-n).png | Doit être clairement distingué de tous les autres hitbursts | Requis | - |
 | hit300k(-n).png | Doit être clairement distingué de tous les autres hitbursts | Requis | - |
-| lighting.png | - | Optionnel | - |
 | particle50.png | Doit être inclus avec toutes les autres particules ; ne fonctionnera que si des hitbursts personnalisés sont inclus | Optionnel | 7x7px |
 | particle100.png | Doit être inclus avec toutes les autres particules ; ne fonctionnera que si des hitbursts personnalisés sont inclus | Optionnel | 7x7px |
 | particle300.png | Doit être inclus avec toutes les autres particules ; ne fonctionnera que si des hitbursts personnalisés sont inclus | Optionnel | 7x7px |
 | sliderpoint10.png | Doit être inclus avec sliderpoint30.png ; ne fonctionnera que si le skin spécifique à l'utilisateur prend en charge le Skin V1 | Optionnel | - |
 | sliderpoint30.png | Doit être inclus avec sliderpoint10.png ; ne fonctionnera que si le skin spécifique à l'utilisateur prend en charge le Skin V1 | Optionnel | - |
 
+### Set des lumières
```
Comment (Walavouchey) [inline wiki/Ranking_Criteria/Skin_Set_List/fr.md:41]: the setting in-game is translated as "Lueurs de fond"

Comment (Biokax) [inline wiki/Ranking_Criteria/Skin_Set_List/fr.md:41]:
```suggestion
### Lueurs de fond
```

File: wiki/Ranking_Criteria/Skin_Set_List/fr.md [hunk 4]
```diff
@@ -37,13 +32,20 @@ L'ensemble de ce skin set est pertinent pour le gameplay.
...
 | hit300g(-n).png | Doit être clairement distingué de tous les autres hitbursts | Requis | - |
 | hit300k(-n).png | Doit être clairement distingué de tous les autres hitbursts | Requis | - |
-| lighting.png | - | Optionnel | - |
 | particle50.png | Doit être inclus avec toutes les autres particules ; ne fonctionnera que si des hitbursts personnalisés sont inclus | Optionnel | 7x7px |
 | particle100.png | Doit être inclus avec toutes les autres particules ; ne fonctionnera que si des hitbursts personnalisés sont inclus | Optionnel | 7x7px |
 | particle300.png | Doit être inclus avec toutes les autres particules ; ne fonctionnera que si des hitbursts personnalisés sont inclus | Optionnel | 7x7px |
 | sliderpoint10.png | Doit être inclus avec sliderpoint30.png ; ne fonctionnera que si le skin spécifique à l'utilisateur prend en charge le Skin V1 | Optionnel | - |
 | sliderpoint30.png | Doit être inclus avec sliderpoint10.png ; ne fonctionnera que si le skin spécifique à l'utilisateur prend en charge le Skin V1 | Optionnel | - |
 
+### Lueurs de fond
```
Comment (DoryanR) [inline wiki/Ranking_Criteria/Skin_Set_List/fr.md:41]:
```suggestion
### Set des lueurs de fond
```

Restons cohérent avec le reste de la page au niveau des titres

**Review by DoryanR — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**